### ISSUE TITLE
Plugin Maintenance

### ISSUE THREAD

**Issue by ts0unami** [parisholley/wordpress-fantastic-elasticsearch#156]
Hi, this plugin hasn't been updated for years now... What is the situation?

Also could I install Elasticsearch on Openshift and then connect that with Wordpress on my server?

How could I do that?

many thanks!
Andrew

**Comment by kaizokou** [comment]
Yes it seems that this plugin is dead. It's a shame as it's seems to be the best available so far. I wanted to use it for my project, but I resign myself to use ElacticPress
